packages/lodestar/test/unit/chain/stateCache/stateContextCheckpointsCache.test.ts

```diff
@@ -0,0 +1,31 @@
+import {config} from "@chainsafe/lodestar-config/default";
+import {BenchmarkRunner} from "@chainsafe/lodestar-utils/test_utils/benchmark";
+import {ssz} from "@chainsafe/lodestar-types";
+import {generateCachedState} from "../../../utils/state";
+import {CheckpointStateCache} from "../../../../src/chain/stateCache";
+
+describe("CheckpointStateCache perf tests", function () {
+  this.timeout(0);
+
+  it("should run test", async () => {
+    const checkpointStateCache = new CheckpointStateCache(config);
+    const state = generateCachedState();
+    const checkpoint = ssz.phase0.Checkpoint.defaultValue();
+    const runner = new BenchmarkRunner("CheckpointStateCache perf test", {
+      maxMs: 999999 * 1000,
```

Comment:
Why 999999 sec? You are telling the benchmark to run for 11 days if necessary in order to complete the 64 runs. You should set minMs to 1000 ms, runs to 1024, otherwise there's to much variability

Reply:
when i init the BenchmarkRunner with these options:
```
const runner = new BenchmarkRunner("CheckpointStateCache perf test", {
      minMs: 1000,
      runs: 1024,
    });
```

I get these results:
```
this PR:
  CheckpointStateCache: add, get, delete                                 21211.61 ops/s      47.14400 us/op  20930 runs    1.002 s
master:
  CheckpointStateCache: add, get, delete                                 13629.18 ops/s      73.37200 us/op  13501 runs    1.002 s
```

It doesn't do the number of runs that I ask it, but the total run time (1.002 s) is consistent between both.  Is this expected?

Reply:
(my original options for BenchmarkRunner had the same thing happen, but still just wondering if that's expected)